tests/bluetooth/bsim_bt/bsim_test_audio/src/unicast_client_test.c

```diff
@@ -398,12 +398,12 @@ static void test_main(void)
 	for (unsigned int i = 0U; i < iterations; i++) {
 		printk("\n########### Running iteration #%u\n\n", i);
 
+		printk("Creating unicast group\n");
+		create_unicast_group(&unicast_group);
```

Comment:
Keep in mind that until `CONFIG_BT_CTLR_CENTRAL_ISO` gets enabled, creating the group here doesn't do anything. 

Have you verified your changes by running the sample?

Reply:
Yes, I run this PR with https://github.com/zephyrproject-rtos/zephyr/pull/53450 fix